monitoring-dashboard/components/org.wso2.micro.integrator.dashboard.web/web-app/src/commons/sideDrawers/DataSourcesSideDrawer.js

```diff
@@ -0,0 +1,112 @@
+/*
+ * Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ *
+ *
+ */
+
+import React from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import Paper from '@material-ui/core/Paper';
+import Box from '@material-ui/core/Box';
+import Grid from '@material-ui/core/Grid';
+import { Table, TableCell, TableRow } from '@material-ui/core';
+import HeadingSection from './commons/HeadingSection'
+import SourceViewSection from './commons/SourceViewSection'
+import Typography from '@material-ui/core/Typography';
+
+export default function DataSourcesSideDrawer(props) {
+    const { nodeData } = props;
+    const nodeId = nodeData.nodeId;
+    const artifactName = nodeData.details.name;
+    const classes = useStyles();
+
+    return (
+        <div className={classes.root}>
+            <Grid container spacing={3}>
+                <Grid item xs={12}>
+                    <HeadingSection name={artifactName} nodeId={nodeId} />
+                    <SourceViewSection
+                        artifactType="data-sources" artifactName={artifactName} nodeId={nodeId}
+                        designContent={<>
```

Comment:
Can we move this section to a new function to improve the readability of the code?

Reply:
Fixed with 795cc4d